src/plone/restapi/tests/test_documentation.py

```diff
@@ -1,12 +1,12 @@
 from base64 import b64encode
 from datetime import datetime
 from unittest.mock import patch
-from pkg_resources import parse_version
 from pkg_resources import resource_filename
 from plone import api
 from plone.app.discussion.interfaces import IConversation
 from plone.app.discussion.interfaces import IDiscussionSettings
 from plone.app.discussion.interfaces import IReplies
+from plone.app.multilingual.interfaces import ITranslationManager
```

Comment:
I think the `plone.app.multilingual` import should be still in a try/except ImportError block, but maybe I am wrong.